pkg/cfn/manager/cluster.go

```diff
@@ -112,6 +109,12 @@ func (c *StackCollection) AppendNewClusterStackResource(plan bool) (bool, error)
 		currentTemplate, iterErr = sjson.Set(currentTemplate, path, value.Value())
 		return iterErr == nil
 	}
+
+	var (
+		addResources []string
```

Comment:
Don't these need to be initialized?

Reply:
Good point, but no need to initialise them for `append` to work: https://play.golang.org/p/_V-VlvaVTac.